jenkins/main.groovy

```diff
@@ -135,8 +136,52 @@ pipeline {
                         description = "Some stages failed"
                     }
                     githubNotify context: prStatusContext, status: status, description: description
+                    notifyBuild(currentBuild.currentResult)
```

Comment:
Same thing here, we want to run this outside of PRs, so move it just after line 140 to be outside the `if (isPullRequest)` conditional